chainer/links/connection/bilinear.py

```diff
@@ -23,14 +22,20 @@ class Bilinear(link.Link):
             Shape of this argument must be
             ``(left_size, right_size, out_size)``. If ``None``,
             :math:`W` is initialized by centered Gaussian distribution properly
-            scaled according to the dimension of inputs and outputs.
+            scaled according to the dimension of inputs and outputs. May also
+            be a callable that takes in a tuple of the matrix shape and returns
+            an initialization.
         initial_bias (tuple): Intial values of :math:`V^1`, :math:`V^2`
             and :math:`b`. The length this argument must be 3.
             Each element of this tuple must have the shapes of
             ``(left_size, output_size)``, ``(right_size, output_size)``,
             and ``(output_size,)``, respectively. If ``None``, :math:`V^1`
             and :math:`V^2` is initialized by scaled centered Gaussian
-            distributions and :math:`b` is set to :math:`0`.
+            distributions and :math:`b` is set to :math:`0`. May also be
+            a callable that takes in a tuple of the matrix shape and
+            returns an initialization, in which case :math:`b` is set to
+            :math:`0` but :math:`V^1` and :math:`V^2` will be initialized
+            from the callable.
```

Comment:
How about passing a 3-tuple of initializers to `initial_bias` and calling `init_weight` with each initializer. It works even if the tuple consists of `ndarry`s because `init_weight` can accept `ndarry`s as an argument `initWith`. Further it makes initialization of `b` more flexible.